awstest/modules/ddb/module.ftl

```diff
@@ -0,0 +1,404 @@
+[#ftl]
+
+[@addModule
+    name="ddb"
+    description="Testing module for the aws ddb component"
+    provider=AWSTEST_PROVIDER
+    properties=[]
+/]
+
+
+[#macro awstest_module_db ]
+
+    [#-- Base database setup --]
+    [@loadModule
+        blueprint={
+            "Tiers" : {
+                "db" : {
+                    "Components" : {
+                        "docdbbase" : {
+                            "Type" : "ddb",
+                            "deployment:Unit" : "aws-docdb-base",
+                            "EngineVersion" : "4.0",
+                            "Port": "mongodb",
+                            "Profiles" : {
+                                "Testing" : [ "docdbbase" ]
+                            },
+                            "GenerateCredentials": {
+                                "Enabled": true,
+                                "EncryptionScheme": "base64"
+                            },
+                            "Settings" : {
+                                "MASTER_USERNAME" : {
+                                    "Value" : "testUser"
+                                },
+                                "MASTER_PASSWORD" : {
+                                    "Value" : "testPassword"
```

Comment:
That test seems mutually exclusive. Generated credentials with fixed values in Settings?

Reply:
Done